test/libsolidity/semanticTests/functionCall/inheritance/call_base_base_explicit.sol

```diff
@@ -0,0 +1,31 @@
+contract BaseBase {
+	function f(uint n) public virtual returns (uint) {
+		return 2 * n;
+	}
+
+	function s(uint n) public returns (uint) {
+		return 4 * n;
+	}
+}
+
+contract Base is BaseBase {
+	function f(uint n) public virtual override returns (uint) {
+		return 3 * n;
+	}
+}
+
+contract Child is Base {
+	function g(uint n) public returns (uint) {
+		// calling base-bse function of a virtual overridden function.
+		return BaseBase.f(n);
+	}
+
+	function k(uint n) public returns (uint) {
+		// Calling base-base function of a non-virtual function.
+		return BaseBase.s(n);
+		// FIXME: Why is Base.s(n) not allowed? Intuitively I'd expect that one to work.
```

Comment:
Probably because it's `= super`. But I agree.

Reply:
Because `s` is not defined in `Base`. Access through base classes resolves to the actual function without virtual function or inheritance lookup.

Reply:
@chriseth I just had a private chat with @ekpyron about it. I am feeling convinced. Thanks. :-)